tests/conftest.py

```diff
@@ -467,16 +490,16 @@ def aperture_scatterguard(done_status):
             AperturePositions.ROBOT_LOAD,
         ),
     )
-    ap_sg.aperture.z.user_setpoint.sim_put(2)  # type: ignore
-    ap_sg.aperture.z.motor_done_move.sim_put(1)  # type: ignore
+    ap_sg.aperture.z.user_setpoint._backend._set_value(2)  # type: ignore
+    ap_sg.aperture.z.motor_done_move._backend._set_value(1)  # type: ignore
```

Comment:
Should: I think you can just set `patch_async_motor(ap_sg.aperture.z, 2),` below and then remove these 2 lines